src/chunkserver/chunk_service.cpp

```diff
@@ -65,8 +66,15 @@ void ChunkServiceImpl::DeleteChunk(RpcController *controller,
                                                response,
                                                done);
     CHECK(nullptr != closure) << "new chunk service closure failed";
-
     brpc::ClosureGuard doneGuard(closure);
+    // auth
+    if (!Authenticator::GetInstance().VerifyCredential(
```

Comment:
it's better to verify token before create `closure`, and also you can also use macro to wrap these code

Reply:
fixed